packages/swing-store/test/util.js

```diff
@@ -0,0 +1,26 @@
+import { Buffer } from 'node:buffer';
+import tmp from 'tmp';
+import { createSHA256 } from '../src/hasher.js';
+
+/**
+ * @param {string} [prefix]
+ * @returns {Promise<[string, () => void]>}
+ */
+export const tmpDir = prefix =>
```

Comment:
It looks like we've written this function a few times. Can you move it to `@agoric/internal` and maybe TODO de-duplication?

Reply:
They're all slightly different. I don't think we should abstract this in this PR. Maybe open an issue instead?

Reply:
#8109